slothy/targets/arm_v81m/arch_v81m.py

```diff
@@ -1493,6 +1501,20 @@ def make(cls, src):
         return obj
 
 
+class vstr_scatter_stored(MVEInstruction):
+    pattern = "vstrw.<dt> <Qd>, [<Rn>, <Qm>, UXTW <imm>]"
```

Comment:
The UXTW should be optional, please add both variants.
Do they exist with byte and halfword datatypes as well? If yes, please add them.